draft-enghardt-panrg-path-properties.md

```diff
@@ -46,8 +48,25 @@ This information is expressed as properties of paths between two endpoints.
 # Introduction
 
 Because the current Internet provides an IP-based best-effort bit pipe, endpoints have little information about paths to other endpoints.
-A Path Aware Network exposes information about one or multiple paths through the network to endpoints,
-so that endpoints can use this information.
+A Path Aware Network exposes information about one or multiple paths through the network to endpoints or the network infrastructure.
+
+It is impossible to provide an exhaustive list of path properties, as with every new technology and protocol, novel properties might become relevant.
+In this document, we specify a set of path properties which might be useful in the following use cases: Traffic policies, network monitoring, and path selection.
+
+- Traffic policies: Network operators or end users can define traffic policies leveraging path awareness.
+Such policies can allow or disallow sending traffic over specific networks or nodes, select an appropriate protocol depending on the capabilities of the on-path devices, or adjust protocol parameters to an existing path.
+An example of a traffic policy is a video streaming application choosing an (initial) video quality based on the achievable data rate, or the monetary cost of the link using a volume-based or flat-rate cost model.
+Another example is an enterprise network where all traffic has to go through a firewall, in which case the endpoint needs to be aware of on-path firewalls.
+
+- Network monitoring: Network operators can use path properties, (e.g., measured by on-path devices), to check whether recent end-user traffic satisfies specific Quality of Service (QoS) requirements, and identify potential problems with their network early on, before the end-user complains.
+
+- Path selection: In some cases, entities can choose to use a certain path (or subset of paths) from a set of paths to achieve a specific goal.
+The possible benefits of a well chosen path varies based on the goal, and thus the only guarantee that a path aware network should give is to not perform worse than the default case without path selection.
+Depending on the goal, paths with different properties are preferred, e.g., retrieving a webpage as quickly as possible requires low latency paths, retrieving a large file in a peer-to-peer network requires paths with high achievable data rate.
```

Comment:
Did you mean "Additionally, there may be trade-offs between path properties, e.g., latency and data rate, and ~~path elements~~ **entities** may influence these trade-offs with their choices."?

Or am I misunderstanding what is meant by trade-offs?
My understanding is: Entity selecting optimal path w.r.t. property A may need to make compromises on property B.

Reply:
I meant that, for example, if I choose a path because it's low latency and then transmit a lot of data over it, it might no longer be low latency.
Also I'm thinking about what Gorry said at the IETF 103 session: On the "Dynamic properties" slide, he commented that "you can trade any of these properties for another", and something about modems. To be honest, I'm not sure I fully understand what he was saying and why you can trade all these properties for each other... but that's where I'm trying to go.
As it's not entirely clear, I'm okay with leaving it out for now and see if we get a better understanding of these trade-offs.